### PR TITLE
[bn] Localize `content/bn/docs/tasks/tools/_index.md`

### PR DESCRIPTION
**Pull request by Imtiaz1234**
Signed-off-by: Saber Imtiaz  saberimtiazmitul@gmail.com
Localizes `content/bn/docs/tasks/tools/_index.md` to Bengali
This is an intial PR as may need some improvements!
Tracking issue: #31677

### CODE REVIEW

File: content/bn/docs/tasks/tools/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: "টুল ইনস্টল করুন"
+description: আপনার কম্পিউটারে Kubernetes টুল সেট আপ করুন।
```
Comment (mitul3737) [inline content/bn/docs/tasks/tools/_index.md:3]: Need to have কুবারনেটিস here

**Review by mitul3737 — CHANGES_REQUESTED** [review]
কুবারনেটিস

**Review by mitul3737 — APPROVED** [review]
LGTM

**Comment by noor-e-jannatnafia** [comment]
Signed-off-by: Noor E Jannat Nafia noorejannatnafia.006@gmail.com
Localizes content/bn/docs/tasks/tools/included/verify-kubectl.md to Bengali
This is an initial PR as may need some improvements!
Tracking isse: 31677

**Pull request merged**